rpp-charter.md

```diff
@@ -37,7 +37,9 @@ New functionalities, not having any equivalents in EPP, may be also defined for
 
 The REGEXT working group is chartered to maintain and standardize extensions to EPP. Consequently, 
 any extensions or changes to EPP, including those related to RPP functionality that do not exist in EPP, 
-are explicitly out of scope for the RPP working group.
+are explicitly out of scope for the RPP working group. It is expected that the REGEXT working group
+will continue with new work on EPP, and the activities of the RPP working group are not intended to
```

Comment:
should " the activities of the RPP working group" not be changed into: "the results of the RPP working group"

if however you do mean activities then. should you not change: "the activities of the RPP working group are not intended to replace EPP"
into : "the activities of the RPP working group are not intended to replace the REGEXT working group"

Reply:
Working groups come and go. EPP started in PROVREG, went to EPPEXT and is now in REGEXT. Arrangement of working group is an IESG matter.

What we want to say is that EPP is not going anywhere and the existence of the RPP working group does not signal the demise of EPP. So I think "activities" is the right word.

Reply:
I do not like this text proposal for 2 reasons:
1) talking about what REGEXT is going to work on seems inappropriate for a new WG charter and not really related to the issue #22 
2) I'm also not fully happy that the second part does tell more about what RPP is not going to do in relation to EPP, rather than saying what is RPP actually going to do. I made a new proposal #43, which in my eyes puts it into the right context.